src/App.css

```diff
@@ -44,7 +44,7 @@ ul.chartLegend {
 .noticeBox {
   border: solid .5px lightgrey;
   margin-left: 5px;
-  padding: 0px 5px;
+  padding: 0 5px;
```

Comment:
You can omit the units when the value is `0`.

https://stylelint.io/user-guide/rules/list/length-zero-no-unit/